### PR TITLE
Test only on latest version on keyvault

### PR DESCRIPTION
**Pull request by sima-zhu**
KeyVault is timeout because we test multiple service versions.
Working with Eng team to figure out whether we could setup the test var though matrix.
Disable the version for now.

### CODE REVIEW

File: sdk/keyvault/tests.yml [hunk 1]
```diff
@@ -8,12 +8,18 @@ jobs:
...
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AZURE_KEYVAULT_ENDPOINT: $(AZURE_KEYVAULT_ENDPOINT)
+      Matrix:
+        Linux - Java 8:
+            OSName: 'Linux'
+            OSVmImage: 'ubuntu-16.04'
+            JavaVersion: '1.8'
+            DisplayName: 'Linux - Java 8'
+            AZURE_TEST_HTTP_CLIENTS: NettyAsyncHttpClient
+            AZURE_KEYVAULT_TEST_CERTIFICATE_SERVICE_VERSIONS: 7.0
```
Comment (g2vinay) [inline sdk/keyvault/tests.yml]: Assuming this is to be set to latest version, 
latest version is this: 7.1-preview

Comment (sima-zhu) [reply sdk/keyvault/tests.yml]: The tests are failing. Try to see whether we need to add quote for env value

File: sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/TestHelper.java [hunk 2]
```diff
@@ -63,6 +63,6 @@ private static boolean shouldServiceVersionBeTested(ConfigurationServiceVersion
         }
         String[] configuredServiceVersionList = SERVICE_VERSION_FROM_ENV.split(",");
         return Arrays.stream(configuredServiceVersionList).anyMatch(configuredServiceVersion ->
-            serviceVersion.toString().equals(configuredServiceVersion.trim()));
+            serviceVersion.getVersion().equals(configuredServiceVersion.trim()));
```
Comment (JonathanGiles) [inline sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/TestHelper.java]: This won't work - you're comparing a ServiceVersion enum value to a string.

Comment (sima-zhu) [reply sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/TestHelper.java]: I am comparing serviceVersion string value to env value.
If say,
```
    V7_0("7.0"),
    V7_1_PREVIEW("7.1-preview");
```
`set AZAZURE_KEYVAULT_TEST_CERTIFICATE_SERVICE_VERSION = 7.0` works

**Pull request merged**